chalicelib_smaht/check_setup.json

```diff
@@ -4,15 +4,15 @@
         "group": "Elasticsearch Checks",
         "schedule": {
             "manual_checks": {
-                "<env-name>": {
+                "data": {
```

Comment:
This is kind of weird going from template-style to a wired name (and even weirder that it uses fourfront names for things, though I guess that's already done).  I'm assuming you've thought this through fairly carefully, but the fact of non-template was something I thought I should at least flag.